code-challenges.js

```diff
@@ -3,7 +3,22 @@
 
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+let iterations = 10;
 
+const fibTaco = (iter) => {
+    let a = 1;
+    let b = 0;
+    let array = []
+    for(let i = 0; i < iter; i++){
+        array.push(a);
+        let temp = a;
+        a = a + b;
+        b = temp;
+    }
+    return array;
+}
```

Comment:
this code works great! you just have to be really careful using variables that are not descriptive. in the case that you'd like to expand your code or communicate your intention, having descriptive variables can really help. when i was reviewing this, i had to manually write out each iteration of the loop and all of the variable re-assignments since it wasn't totally clear what your logic was. comments can help with this too.